invenio_app_rdm/config.py

```diff
@@ -670,3 +670,5 @@ def files_rest_permission_factory(obj, action):
 RDM_REQUESTS_ROUTES = {
     'user-dashboard-community-submission': '/me/requests/<pid_value>',
 }
+
+RDM_REQUESTS_CONVERSATION_PAGE_SIZE = 15
```

Comment:
sorry, naming police.
this module does not know anything about conversation tab, in the backend it is called "timeline" everywhere so lets keep it consistent. Additionally, shouldn't this variable be defined in invenio-requests?
```suggestion
RDM_REQUESTS_TIMELINE_PAGE_SIZE = 15
```

Reply:
Move to `invenio-requests`? Why here?
```suggestion
REQUESTS_TIMELINE_PAGE_SIZE = 15
```
Please also add a docstring with explanation